src/core/Akka/Util/WildcardTree.cs

```diff
@@ -15,8 +15,10 @@ namespace Akka.Util
     /// A searchable nested dictionary, represents a searchable tree structure underneath
     /// </summary>
     /// <typeparam name="T">TBD</typeparam>
-    public sealed class WildcardTree<T> where T:class
```

Comment:
Meh.... This is a breaking change but this really never should have been part of the public API

Reply:
The original class from the scala implementation was private, it was never ment to be public in the first place.